### ISSUE TITLE
PROBLEM with Fake data

### ISSUE THREAD

**Issue by drjakob** [roschaefer/story.board#178]
How to generate Fake Data for all sensors (or set actual dates for all of them).

**Comment by drjakob** [comment]
A button "actualize fake data" or an automated function with every preview could be a solution.

**Comment by roschaefer** [comment]
Could you please explain the problem here? Do you want to generate fake data for all sensors at once in order to save time and hassle?

**Comment by drjakob** [comment]
Yes,  I want to generate fake data for all sensors at once. I think I would prefer an automated generation of one new datapoint for each sensor. (Because otherwise every test would need ten minutes of generating data.)

**Comment by roschaefer** [comment]
Given there is a valid range for every sensor (you specify that if you generate fake data at the moment) I could implement it. But what on earth is the possible use case? It seems you just want to have a button `change everything`. You cannot see what has changed, what are the generated values. This feature does not help to debug the configuration. The only thing that comes to my mind is that you notice possible contradictions between some text components for a certain state of the system.

**Comment by drjakob** [comment]
Use Case: I want test the text generator NOW. So I need ACTUAL fake data, otherwise it won't produce any text result. Correct me if I am wrong?

**Comment by roschaefer** [comment]
For every sensor you can create fake data. If the triggers and text components are set up correctly, you should see the result in the preview of the report. The only thing that you cannot test are `from` and `to` days of the text component. But you could just leave them out, and add them as soon you are confident enough.

**Comment by drjakob** [comment]
As a reporter I want to test my report at different moments in time
because I have to see how the text components (of different authors!) will work together.
